<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wake Up</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>290</width>
      <height>281</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>MAC-Adressen</string>
     </attribute>
     <widget class="QComboBox" name="mainListMacs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>281</width>
        <height>32</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="mainButtonSendPacket">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>221</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Gerät einschalten</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="mainLabelMacAddress">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MAC-Adresse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainButtonDelete">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Eintrag löschen</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addMacTab">
     <property name="whatsThis">
      <string/>
     </property>
     <attribute name="title">
      <string>Hinzufügen</string>
     </attribute>
     <widget class="QLabel" name="addMacLabelName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="addMacLabelMacAddress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MAC-Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addMacLineName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addMacLineMacAddress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addMacButtonSave">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
     <widget class="QLabel" name="addMacLabelInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Fehler</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
